main.css

```diff
@@ -0,0 +1,367 @@
+body{
```

Comment:
As I mentioned in my main comment, it is possible to refer to elements within different HTML files in a single CSS. This can be achieved by putting the elements you want to edit in the same class, but giving them unique IDs on each HTML page to refer back to them. On your yelp.html, you could do something like...

```html
<body id = "yelp"> <div class = "mainDiv">`
```
and then do something similar on the index.html..
```html
<body id = "home"> <div class = "mainDiv">
```
and then finally, refer back to them on a single CSS..
```css
#yelp .mainDiv {
border: 1px solid green;
width: 200px;
}

#home .mainDiv {
border: 1px solid red;
width: 100px;
}